[8th May 2024] Add deprecation notices to CodeQL for VS Code documentation (#16320)
github/codeql: README.md

## How do I learn CodeQL and run queries?

> There is [extensive documentation](https://codeql.github.com/docs/) on getting started with writing CodeQL using the [CodeQL extension for Visual Studio Code](https://docs.github.com/en/code-security/codeql-for-vs-code/) and the [CodeQL CLI](https://docs.github.com/en/code-security/codeql-cli).

Reviewer: Tiny change to make it clearer that codeql.github.com/docs is now primarily the home for CodeQL language information.

```suggestion
There is extensive documentation about the [CodeQL language](https://codeql.github.com/docs/), writing CodeQL using the [CodeQL extension for Visual Studio Code](https://docs.github.com/en/code-security/codeql-for-vs-code/) and using the [CodeQL CLI](https://docs.github.com/en/code-security/codeql-cli).
```
